Assets/Scripts/Utils/Utilities.cs

```diff
@@ -0,0 +1,18 @@
+using UnityEngine;
+
+public class Utilities
+{
+    // Return number of digits in an int
+    public static int countDigitsInt( int num )
```

Comment:
Huge nit: could replace this function with
```
(int)Math.Log(num, 10) + 1
```

Reply:
You beautiful man
Next version 😆